src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <div class="navigation_logo">
+        <a href="#">
+          <img
+            src="images\logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav_list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a
              href="#"
              class="nav__link"
            >
              Samsung
            </a>
          </li>
          
          <li class="nav__item">
            <a
              href="#"
              class="nav__link"
            >
              Smartphones
            </a>
          </li>
```

it is a good idea to add blank lines between multiline siblings. Pls fix whole list